ldi-orchestrator/ldio-application/src/test/resources/features/management.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Management of pipelines in LDIO
+  As a user
+  I want to add and remove LDIO pipelines
+
+  @creation
+  Scenario: Creating a pipeline with a valid json
```

Comment:
- Can you add a test case that verifies that the correct exception is thrown when you add a pipeline that already exists?
- Can you add a test case that verifies that you can add more than 1 pipeline?
- Can you test the Get endpoint as well?